app/models/concerns/fog_extensions/ovirt/template.rb

```diff
@@ -0,0 +1,13 @@
+module FogExtensions
+  module Ovirt
+    module Template
+      def full_name
+        "#{name}#{full_version.present? ? " (#{full_version})" : nil}"
```

Comment:
I'd prefer complex coding outside of strings interpolation, but this is solely my opinion. :-)